src/Pages/SignUp.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Nit: Let's rename this folder from `Pages` to `pages` since all of the other folders are lowercase.

Reply:
Once this is resolved, I think the changes should be good to go 👍

Reply:
hmm, its renamed on my end, i dont know why it doesn't update?

Reply:
Oh ok, it should be fine then!